run.go

```diff
@@ -66,6 +66,10 @@ func runflags() []cli.Flag {
 			Name:  "no-cache",
 			Usage: "Don't use Docker cache for the build",
 		},
+		cli.StringFlag{
```

Comment:
not sure if `fn call` has separate flag set, i only see this flag added here. (just checking)